### PR TITLE
[Fixes #976] Fix missing required accessToken for MapBox background when adding a map to a geostory

### LINKED ISSUE #976: Fix missing required accessToken for MapBox background when adding a map to a geostory
Fix missing required accessToken for MapBox background when adding a map to a geostory

### PR DESCRIPTION
**Pull request by allyoucanmap**
Update MapStore submodule to 2022.01.xx branch with complete support for mapbox accessToken

### CODE REVIEW

**Review by DavidQuartz — APPROVED** [review]

File: geonode_mapstore_client/client/package.json [hunk 1]
```diff
@@ -26,7 +26,7 @@
   "author": "GeoSolutions",
   "license": "BSD-2-Clause",
   "devDependencies": {
-    "@mapstore/project": "1.0.21",
+    "@mapstore/project": "1.0.22",
```
Comment (allyoucanmap) [inline geonode_mapstore_client/client/package.json:29]:
```suggestion
    "@mapstore/project": "1.0.23",
```

**Comment by giohappy** [comment]
@allyoucanmap it looks good.

image

**Comment by giohappy** [comment]
it works

image

**Comment by giohappy** [comment]
@allyoucanmap this was ported to 4.x right?

**Comment by allyoucanmap** [comment]
@giohappy backported and merged on 4.x

**Pull request merged**